deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -646,9 +646,6 @@ cephObjectStores:
 # cephECStorageClass also is disabled by default, please remove the comments and set desired values to enable it
 # if cephECBlockPools are uncommented you must remove the comments of cephEcStorageClass as well
 #cephECStorageClass:
-#  name: rook-ceph-block
```

Comment:
I think you need to revert this line, did you only intend to remove lines 650-651?

Reply:
Yes correct, i meant only to delete provisioner name